cmd/containerd/command/config.go

```diff
@@ -113,3 +117,42 @@ var configCommand = cli.Command{
 		},
 	},
 }
+
+func platformAgnosticDefaultConfig() *srvconfig.Config {
+	return &srvconfig.Config{
+		Version: 1,
+		Root:    defaults.DefaultRootDir,
+		State:   defaults.DefaultStateDir,
+		GRPC: srvconfig.GRPCConfig{
+			Address: defaults.DefaultAddress,
+		},
+		DisabledPlugins:  []string{},
+		RequiredPlugins:  []string{},
+		StreamProcessors: streamProcessors(),
+	}
+}
+
+func streamProcessors() map[string]srvconfig.StreamProcessor {
+	const (
+		ctdDecoder = "ctd-decoder"
+		basename   = "io.containerd.ocicrypt.decoder.v1"
+	)
+	decryptionKeysPath := filepath.Join(defaults.DefaultConfigDir, "imgcrypt", "keys")
```

Comment:
Bike shedding: should this be "imgcrypt" or "ocicrypt"?

Reply:
imgcrypt is the project where ctd-decorder is coming from but ocicrypt could be a more generic name for a directory for looking for keys used generally for image encryption. -> "ocicrypt" ?

Reply:
Changed to `ocicrypt`